modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/ObjectValidation/BasicInfo.tsx

```diff
@@ -170,7 +170,7 @@ export function BasicInfo({
 											objectValidationRuleSettings: [
 												{
 													name:
-														'objectFieldExternalReferenceCode',
+														'outputObjectFieldExternalReferenceCode',
```

Comment:
@naraandrade Can we make it a constant?
It's used in three differrent places.

Reply:
@mateussgn Sure! You can create a constant for it inside https://github.com/liferay/liferay-portal/blob/master/modules/apps/object/object-web/src/main/resources/META-INF/resources/js/utils/constants.ts.

Reply:
I'll do it!
Thanks for the tip!